framework/src/auxkernels/AuxKernel.C

```diff
@@ -194,20 +199,24 @@ AuxKernel::compute()
       _local_ke.zero();
 
       // assemble the local mass matrix and the load
-      for (unsigned int i = 0; i < _test.size(); i++)
-        for (_qp = 0; _qp < _qrule->n_points(); _qp++)
+      for (_qp = 0; _qp < _qrule->n_points(); _qp++)
```

Comment:
No I did not do the timing. I assumed `computeValue()` could be very costly. It also could surprise developers when they see more local evaluations than the number of local quadrature points.